tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -399,13 +397,13 @@ export class GraphExecutor implements FunctionExecutor {
       this.checkOutputs(outputs);
     }
 
-    // For model debug.
+    // Keep tensors if KEEP_INTERMEDIATE_TENSORS is on.
     try {
-      this.keepTensorForDebug = env().getBool('KEEP_INTERMEDIATE_TENSORS');
+      this.keepIntermediateTensors = env().getBool('KEEP_INTERMEDIATE_TENSORS');
     } catch (e) {
+      this.keepIntermediateTensors = false;
       console.warn(e.message);
     }
```

Comment:
This is great, thanks. Done

Reply:
True, the last call's tensors will still be leaked. However, this change will limit the amount of leaking that's possible, so a model can only ever leak however many intermediate tensors it has (and not more).

I wouldn't be opposed to removing this, though, especially since it makes it hard to collect intermediate tensors from multiple runs. I'll leave it up to you.

Reply:
If you do end up removing it, make sure to remove it from `execute()` as well.